nautobot/utilities/filters.py

```diff
@@ -234,6 +296,13 @@ class BaseFilterSet(django_filters.FilterSet):
             models.URLField: {"filter_class": MultiValueCharFilter},
             models.UUIDField: {"filter_class": MultiValueCharFilter},
             MACAddressField: {"filter_class": MultiValueMACAddressFilter},
+            # TODO(jathan): This was added for objects to be able to do reverse mapping to their
+            # DynamicGroup membership.
+            # Ref: https://github.com/carltongibson/django-filter/issues/1107
+            models.JSONField: {
```

Comment:
Possibly move this up so that it's alphabetically inline with the other fields in this dict?